src/statick_md/plugins/tool/writegood_tool_plugin.py

```diff
@@ -0,0 +1,91 @@
+"""Apply writegood tool and gather results.
+
+Website: https://github.com/btford/write-good
+"""
+
+import logging
+import re
+import subprocess
+from typing import List, Match, Optional, Pattern
+
+from statick_tool.issue import Issue
+from statick_tool.package import Package
+from statick_tool.tool_plugin import ToolPlugin
+
+
+class WriteGoodToolPlugin(ToolPlugin):  # type: ignore
+    """Apply writegood tool and gather results."""
+
+    def get_name(self) -> str:
+        """Get name of tool."""
+        return "writegood"
+
+    def scan(self, package: Package, level: str) -> Optional[List[Issue]]:
+        """Run tool and gather output."""
+        tool_bin = "writegood"
```

Comment:
this seems to work, but the package's documentation uses `write-good` for cli calls. I think it would be good to match the documentation.